noxfile.py

```diff
@@ -0,0 +1,127 @@
+import nox
+import os
+import sys
+
+
+@nox.session(name="pybamm-requires")
+def run_pybamm_requires(session):
+    homedir = os.getenv("HOME")
+    session.env["SUNDIALS_INST"] = session.env.get("SUNDIALS_INST", f"{homedir}/.local")
+    session.env[
+        "LD_LIBRARY_PATH"
+    ] = f"{homedir}/.local/lib:{session.env.get('LD_LIBRARY_PATH')}"
+    if sys.platform != "win32" or sys.platform != "darwin":
+        session.install("wget", "cmake")
+        session.run("python", "scripts/install_KLU_Sundials.py")
+        session.run(
+            "git",
+            "clone",
+            "https://github.com/pybind/pybind11.git",
+            "pybind11/",
+            external=True,
+        )
+
+
+@nox.session(name="coverage")
+def run_coverage(session):
+    homedir = os.getenv("HOME")
+    session.env["SUNDIALS_INST"] = session.env.get("SUNDIALS_INST", f"{homedir}/.local")
+    session.env[
+        "LD_LIBRARY_PATH"
+    ] = f"{homedir}/.local/lib:{session.env.get('LD_LIBRARY_PATH')}"
+    session.install("coverage")
+    session.run("pip", "install", "-e", ".")
+    if sys.platform != "win32" or sys.platform != "darwin":
+        session.install("scikits.odes")
+        session.run("pybamm_install_jax")
+    session.run("coverage", "run", "--rcfile=.coveragerc", "run-tests.py", "--nosub")
+    session.run("coverage", "combine")
+    session.run("coverage", "xml")
+
+
+@nox.session(name="integration", reuse_venv=True)
+def run_integration(session):
+    homedir = os.getenv("HOME")
+    session.env["SUNDIALS_INST"] = session.env.get("SUNDIALS_INST", f"{homedir}/.local")
+    session.env[
+        "LD_LIBRARY_PATH"
+    ] = f"{homedir}/.local/lib:{session.env.get('LD_LIBRARY_PATH')}"
+    session.run("pip", "install", "-e", ".")
```

Comment:
```suggestion
    session.run("pip", "install", "-e", ".[dev]")
```

Reply:
Also, looking at this again, it would be better to just replace all the `session.run("pip", "install", "-e", "something")` occurrences with `session.install("-e", "something")`.

Reply:
Sure, doing it this way.